### PR TITLE
added PyPI upload docs

### LINKED ISSUE #21: add instructions for updating pypi libraries
I have my own notes about how to push updates to PyPI. They probably should be added to these instructions.

### PR DESCRIPTION
**Pull request by pdpinch**
#### What are the relevant tickets?

closes #21 

#### What's this PR do?

add instructions for releases to PyPI. 

It does not make any effort to automate the release to PyPI. 

#### How should this be manually tested?

Try following the instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,6 +110,31 @@ For example, to wait for successful deployment on micromasters to RC:
...
+
+1. Get the PyPI credentials from DevOps.
+If you haven't already, you should set up a ``.pypirc`` file as described in 
+http://peterdowns.com/posts/first-time-with-pypi.html
+1. Review the metadata in ``setup.py``
+Most repos should already be set up with proper metadata. If not, consult the documentation at 
+https://docs.python.org/3.6/distutils/setupscript.html#meta-data
+1. Check the version number in ``setup.py``. It should be the same as the release number. 
+
+Do a test run with the pypittest repository:
```
Comment (noisecapella) [inline README.md]: I think there's an extra `t` here

**Comment by pdpinch** [comment]
@mitodl/devops can one you take a look at this and verify my instructions?

**Comment by blarghmatey** [comment]
Aside from the extra `t` :+1:

**Comment by pdpinch** [comment]
@noisecapella can you give me a thumb? (or maybe I can just dismiss your review?)

**Comment by pdpinch** [comment]
nevermind. I just needed to reload the page.

**Pull request merged**